packages/sfui/frameworks/react/hooks/usePopover/usePopover.ts

```diff
@@ -19,8 +19,8 @@ export function usePopover(options: UsePopoverOptions) {
 
   const style = {
     position: strategy,
-    top: y ?? 0,
-    left: x ?? 0,
+    top: y,
+    left: x,
```

Comment:
Question: default values are not needed anymore?

Reply:
now x,y return always number not like previously `null` https://github.com/floating-ui/floating-ui/releases/tag/%40floating-ui%2Freact-dom%402.0.0